lib/client-api/src/args.ts

```diff
@@ -72,7 +72,9 @@ export const combineArgs = (value: any, update: any): Args => {
 export const validateOptions = (args: Args, argTypes: ArgTypes): Args => {
   return Object.entries(argTypes).reduce((acc, [key, { options }]) => {
     if (!options) {
-      acc[key] = args[key];
+      if (key in args) {
```

Comment:
We could also use `args[key] !== undefined`, or various other techniques. I honestly don't know what's the right way.

Reply:
Good catch. I think this is fine and better than checking against undefined. The alternative would be hasOwnProperty, but since we're dealing with plain objects, it doesn't matter.

Reply:
Yeah, I guess technically someone could have an arg called something like `hasOwnProperty` but I suspect they'll be in a world of hurt if they do that anyway.